systemtest/src/main/java/io/strimzi/systemtest/utils/specific/KeycloakUtils.java

```diff
@@ -208,6 +174,30 @@ public static String updatePolicyOfRealmClient(String namespaceName, String base
         ).out();
     }
 
+    /**
+     * Creates Keycloak realm
+     * @param namespaceName namespace name
+     * @param baseURI base uri for accessing Keycloak API
+     * @param token admin token
+     * @param realmData realm data/configuration in JSON format
+     * @return result of creation
+     */
+    public static String createRealm(String namespaceName, String baseURI, String token, String realmData) {
+        String coPodName = kubeClient(namespaceName).getClusterOperatorPodName();
```

Comment:
is it a good idea to use CO pod for this?

Reply:
It is done for all other methods inside `KeycloakUtils` as well 🤷 but we can f.e. create a `Scraper` pod here and do the requests from there

Reply:
But that's maybe for different PR, as it would change a lot of classes again.
WDYT?

Reply:
Yes, different PR makes sense. Maybe some issue for tracking would be useful.

Reply:
https://github.com/strimzi/strimzi-kafka-operator/issues/7892